backend/registration/urls.py

```diff
@@ -1,6 +1,7 @@
-from .views import RegisterAPI
+from .views import RegisterAPI,LoginAPI
 from django.urls import path
 
 urlpatterns = [
     path('api/user/register', RegisterAPI.as_view(), name='register'),
+    path('api/user/login/', LoginAPI.as_view(), name='login'),
```

Comment:
Update to `path('api/user/login', LoginAPI.as_view(), name='login'),`

Reply:
@aika-blanes 
I think its a best practice not to include a forward slash at the end of the api endpoint.

Maybe we can update this to:
`'api/user/login'`

References:
- https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/
- https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/
- https://stackoverflow.com/questions/7140074/restfully-design-login-or-register-resources